features/chat/chat-api-data.ts

```diff
@@ -71,6 +71,39 @@ export const PromptDataGPT = async (props: PromptGPTProps) => {
   return new StreamingTextResponse(stream);
 };
 
+export const simplePromptDataGPT = async (query: string) => {
+  const chatModel = new ChatOpenAI({
+    temperature: 0,
+    streaming: true,
+  });
+
+  const relevantDocuments = await findRelevantDocuments(query);
+
+  const chain = loadQAMapReduceChain(chatModel, {
+    combinePrompt: defineSystemPrompt(),
+  });
+  const { stream, handlers } = LangChainStream();
+
+  const memory = buildMemory([]);
```

Comment:
this could be improved to have access to user thread messages and build memory on it.